subworkflows/local/utils_nfcore_seqinspector_pipeline/main.nf

```diff
@@ -84,7 +84,7 @@ workflow PIPELINE_INITIALISATION {
         .fromSamplesheet("input") // Validates samplesheet against $projectDir/assets/schema_input.json. Path to validation schema is defined by $projectDir/nextflow_schema.json
         .map {
             meta, fastq_1, fastq_2 ->
-                def id_string = "${meta.sample}_${meta.project ?: "ungrouped"}_${meta.lane}"
+                def id_string = "${meta.sample}_${meta.group ?: "ungrouped"}_${meta.lane}"
```

Comment:
It's been removed from required.

Reply:
😳 Have I then been reviewing the wrong/outdated version of this PR all the time? Because I ran `gh pr 13 checkout` and it is still in there for me locally ?!?

Reply:
I removed it so we would be able to run on sequencing platforms without lanes, e.g. ONT.

Reply:
> Doesn't lane need a default value too if it's not required?

The way I understand it from the code in `workflows/seqinspector.nf`, the `filter` operation means it simply doesn't perform lane-wise multiQC if the field is empty.

```
    multiqc_extra_files_per_group = ch_multiqc_files
        .filter { meta, sample -> meta.group }
        .map { meta, sample -> meta.group }
        .unique()
        .map { group -> [group:group] }
        .combine(ch_multiqc_extra_files)
```

Reply:
Sure, but what about the other paths, e.g. channel where `meta.group` has a setting, but `meta.lane` has nothing (and the filter is on `meta.group`)?
The name will include `null` in it.

Reply:
Could you point out where this would be an issue?

I'll note that I don't mind re-working this code into something more explicit, I simply lack the know-how as of now 😆

Reply:
I guess the question is, is `id` used for anything important: For example with the Promethion test, the csv looks like:
```
sample,lane,group,fastq_1,fastq_2,rundir
hg001,,r10p41_e8p2_human_runs_jkw,https://github.com/nf-core/test-datasets/raw/seqinspector/testdata/PromethION/20230505_1857_1B_PAO99309_94e07fab/fastq_pass/PAO99309_pass__94e07fab_c3641428_1.fastq.gz,,
```
and then the id string should be: `hg001_r10p41_e8p2_human_runs_jkw_null`. Does it matter that this is the case?
When you're grouping the files by group `r10p41_e8p2_human_runs_jkw` I guess lane information is not needed at all downstream of this?

Reply:
Thanks for clarifying! From what I remember of the initial meeting, the `id` was intended as simply a concatenation of fields to ensure uniqueness within the pipeline run. This is still ensured even if some of the fields are null, right?

Intuitively I think having a consistent way to generate the `id` that sometimes contains null is preferable to setting up different conventions for generating it across different sequencing platforms.